udata/core/discussions/csv.py

```diff
@@ -0,0 +1,21 @@
+from udata.frontend import csv
+
+from .models import Discussion
+
+
+@csv.adapter(Discussion)
+class DiscussionCsvAdapter(csv.Adapter):
+    fields = (
+        "id",
+        "user",
+        "subject",
+        ("subject_class", "subject.class"),
+        ("subject_id", "subject.id"),
+        "title",
+        ("size", lambda o: len(o.discussion)),
+        ("participants", lambda o: ",".join(set(msg.posted_by.id for msg in o.discussion))),
+        ("messages", lambda o: "\n".join(msg.content.replace("\n", " ") for msg in o.discussion)),
+        "created",
+        "closed",
+        "closed_by",

```

Comment:
It may be worth noting that `closed_by` currently shows the user name (and not the id as in participants). It seems more user-friendly but less automatisation friendly.

Reply:
I had not noticed (and I was not expecting that just mentioning `closed_by` would pick the user name, as it is an object). We could change it for `("closed_by", "closed_by.id"),`, what do you think?

Reply:
I think it comes from https://github.com/opendatateam/udata/blob/5f54568240ed430726bb68e0a307415e9870aa8e/udata/core/user/models.py#L105

Reply:
And yes, let's split the closed by name and id